specification/azurefleet/AzureFleet.Management/tspconfig.yaml

```diff
@@ -37,3 +37,13 @@ options:
     service-name: Compute Fleet
     examples-directory: "examples"
     flavor: azure
+  "@azure-tools/typespec-go":
+    service-dir: "sdk/resourcemanager/computefleet"
+    package-dir: "armcomputefleet"
+    module: github.com/Azure/azure-sdk-for-go/{service-dir}/{package-dir}
+    flavor: "azure"
```

Comment:
```suggestion
```

Reply:
We will keep this as TypeSpec validation tool validate this requirement.